tensorflow/python/autograph/converters/logical_expressions.py

```diff
@@ -49,9 +44,10 @@ def _overload_of(self, operator):
     op_type = type(operator)
     if op_type in LOGICAL_OPERATORS:
       return LOGICAL_OPERATORS[op_type]
-    if self.ctx.user.options.uses(converter.Feature.EQUALITY_OPERATORS):
-      if op_type in EQUALITY_OPERATORS:
-        return EQUALITY_OPERATORS[op_type]
+    if op_type == gast.Eq:
```

Comment:
Oops, I missed this. These need to be removed as well. Basically, don't handle equality operators at all, and led the Tensor overloads handle it.

Reply:
If I remember correctly it doesn't work.

Reply:
Yes, it is going to fail with `TypeError: Can not convert a bool into a Tensor or Operation.`:

https://github.com/tensorflow/tensorflow/blob/5930698e662460a054695dbc68950a98fd751902/tensorflow/python/autograph/converters/logical_expressions_test.py#L26-L34

Reply:
Yes, that test would now require tf2. Adding a call to `tf.compat.v1.enable_tensor_equality()` should fix it.

Reply:
The cache was invalidated again locally "in the container". I've tried to make a blind commit/push.